plugins/UsersManager/lang/en.json

```diff
@@ -4,150 +4,150 @@
-        "ThereAreCurrentlyNRegisteredUsers": "There are currently %s registered users.",
-        "TokenAuth": "API Authentication Token",
-        "TokenRegenerateConfirmSelf": "Changing the API authentication token will invalidate your own token. If the current token is in use, you need to update all API clients with the newly generated token. Do you really want to change your authentication token?",
+        "ThereAreCurrentlyNRegisteredUsers": "%s registered users.",
+        "TokenAuth": "API authentication token",
+        "TokenRegenerateConfirmSelf": "Change your authentication token? This changes the API authentication to invalidate your own token. If the current token is in use, all API clients must be updated with the newly generated token.",
         "TokenRegenerateTitle": "Regenerate",
-        "TokensSuccessfullyDeleted": "All tokens were successfully deleted",
-        "TokenSuccessfullyDeleted": "Token was successfully deleted",
+        "TokensSuccessfullyDeleted": "All tokens deleted",
+        "TokenSuccessfullyDeleted": "Token deleted",
         "DeleteAllTokens": "Delete all tokens",
-        "ExpiredTokensDeleteAutomatically": "Tokens with an expire date will be deleted automatically.",
-        "TokensWithExpireDateCreationBySystem": "Tokens with expire date can currently only be created by the system",
-        "TokenAuthIntro": "Tokens you have generated can be used to access the Matomo reporting API, Matomo tracking API, and exported Matomo widgets and have the same permissions as your regular user login. You can use these tokens also for the Matomo Mobile app.",
+        "ExpiredTokensDeleteAutomatically": "Tokens with an expiration date will be deleted.",
+        "TokensWithExpireDateCreationBySystem": "Tokens with an expiration date can currently only be created by the system",
+        "TokenAuthIntro": "Tokens you have generated can be used to access the Matomo reporting API, Matomo tracking API, exported Matomo widgets and have the same permissions as your regular user login. You can also use these tokens for the Matomo mobile app.",
         "TypeYourPasswordAgain": "Type your new password again.",
         "TypeYourCurrentPassword": "Please type your current password to confirm the password change.",
         "User": "User",
         "UserHasPermission": "%1$s currently has %2$s access for %3$s.",
         "UserHasNoPermission": "%1$s currently has %2$s to %3$s",
-        "UsersManagement": "Users Management",
+        "UsersManagement": "User Management",
         "UsersManagementMainDescription": "Create new users or update the existing users. You can then set their permissions above.",
-        "WhenUsersAreNotLoggedInAndVisitPiwikTheyShouldAccess": "When users are not logged in and visit Matomo, they should initially see",
+        "WhenUsersAreNotLoggedInAndVisitPiwikTheyShouldAccess": "When users are not logged in and visit Matomo, they should initially be met by",
```

Comment:
This one is displayed in this context:
![image](https://user-images.githubusercontent.com/1579355/177499391-18d93147-335e-4b68-8709-69018481f68d.png)

Not sure if the new wording fits there.